homestead.md

```diff
@@ -87,7 +87,7 @@ If this command fails, make sure your Vagrant installation is up to date.
 
 #### Installing Homestead
 
-You may install Homestead by cloning the repository. Consider cloning the repository into a `Homestead` folder within your "home" directory, as the Homestead box will serve as the host to all of your Laravel projects:
+Next, install Homestead itself by cloning the repository. Consider cloning the repository into a `Homestead` folder within your "home" directory, as the Homestead box will serve as the host to all of your Laravel projects:
```

Comment:
remove `itself`, not necessary, and Homestead kind of encompasses many things. This phrasing makes it seem like Homestead is just the repo.

I think `Next` is a good clarification.